afew/MailMover.py

```diff
@@ -72,31 +75,45 @@ def move(self, maildir, rules):
                     if self.dry_run:
                         continue
                     try:
-                        shutil.copy2(fname, self.get_new_name(fname, destination))
-                        to_delete_fnames.append(fname)
-                    except shutil.SameFileError:
-                        logging.warn("trying to move '{}' onto itself".format(fname))
-                        continue
+                        logging.info("Copy original mail: "+fname)
+                        logging.info("Path of mail: "+os.path.dirname(fname))
+                        logging.info("Copy to: "+destination)
+                        dbpath = self.db.get_path()
+                        logging.info("Path of DB: "+dbpath)
+                        relpath=os.path.relpath(os.path.dirname(destination), dbpath)
+                        filename = os.path.basename(fname)
+                        logging.info("filename: "+filename)
+                        logging.info("Relative path DB -> file: " + relpath)
+                        # 1 COPY
+                        shutil.copy2(fname, self.get_new_name(
+                                                            fname,
+                                                            destination))
+                        dest_file_rel = dbpath+'/'+relpath+'/'+filename
```

Comment:
Please use `os.path.join`, or even better [pathlib](https://docs.python.org/3/library/pathlib.html) to join paths.

Reply:
Ok I will upgrade that